ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts

```diff
@@ -0,0 +1,125 @@
+export class CloseDecisionPage
+{
+ 
+	public withSupportingNotes(supportingNotes: string): this {
```

Comment:
This looks like a duplicate of the method below

Reply:
Code refactored